Session/Conversations/Message Cells/VisibleMessageCell.swift

```diff
@@ -22,7 +22,6 @@ final class VisibleMessageCell: MessageCell, TappableLabelDelegate {
     private lazy var authorLabelTopConstraint = authorLabel.pin(.top, to: .top, of: self)
     private lazy var authorLabelHeightConstraint = authorLabel.set(.height, to: 0)
     private lazy var profilePictureViewLeadingConstraint = profilePictureView.pin(.leading, to: .leading, of: self, withInset: VisibleMessageCell.groupThreadHSpacing)
-    private lazy var profilePictureViewWidthConstraint = profilePictureView.set(.width, to: Values.verySmallProfilePictureSize)
```

Comment:
Remove this constraint will end up with a UI bug for incoming messages in 1-1 convos. The leading constraint of content view is based on the trailing of profile picture view, if there is no width of profile picture view, the leading constraint of content view will not work.

Reply:
Ah yep I see it - sorry this change hasn't gone through QA yet but should as User Config is getting tested, I'll push up a fix now (I'll have the `ProfilePictureView` update it's constraints when `isHidden` is updated rather than adding this back since it's managing it's own size constraints now)